test/testground/network/consensus_node.go

```diff
@@ -321,3 +326,10 @@ func parsePeerID(input string) (string, net.IP, int, error) {
 
 	return address, ip, portInt, nil
 }
+
+func safeConvert(x int) (uint16, error) {
```

Comment:
[optional]
Same, would be nice if this is renamed `safeConvertIntToUint16`

Reply:
a9327a7dc995ecd5e76b03ec02f371aa852fc7ec